pkg/controller/clustersync/clustersync_controller.go

```diff
@@ -502,6 +502,13 @@ func (r *ReconcileClusterSync) Reconcile(ctx context.Context, request reconcile.
 	}
 	// Clear the cluster sync failing metric
 	if !failing {
+		// Set the metric to zero first since clearing the metric would only stop recording it.
```

Comment:
I'd like to stress that this is not what we usually do, metrics should always be considered in conjunction with their timestamps, but I also do not see any problem with this - and we might want to consider doing so for other gauges - particularly failure reporting gauges across the codebase

Reply:
As we discussed in scrum, I think "usually" we're tallying a discrete event in the past (e.g. "a provision failed") whereas here we're tracking in realtime how long the clustersync *has been* in the failing state.